app/assets/stylesheets/application.tailwind.css

```diff
@@ -59,15 +68,9 @@
   body {
     @apply bg-background text-foreground;
     font-feature-settings: "rlig" 1, "calt" 1;
-  }
-}
-
-@layer utilities {
-  .font-bold {
-    font-weight: 620;
-  }
 
-  .font-semibold {
-    font-weight: 580;
+    /* docs specific */
+    /* https://css-tricks.com/snippets/css/system-font-stack/ */
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
Shadcn docs uses a [Vercel font ](https://vercel.com/font) called Geist that are best suited for Next.js. We're keeping it simples and using native font for docs